### ISSUE TITLE
interlaced PNG

### ISSUE THREAD

**Issue by sofar** [4Evergreen4/quartz#5]
```
./mods/quartz/textures/quartz_ore.png: PNG image data, 16 x 16, 8-bit/color RGBA, interlaced
```

not all libpng versions can handle interlaced options, this may cause rendering issues in irrlicht.

please convert the texture to non-interlaced PNG.

**Comment by 4Evergreen4** [comment]
I think I fixed the image in commit 791ae9da0655f192bc0e89be59a33151f0df49ae 
Is it fixed now?

**Comment by sofar** [comment]
```
$ file textures/*
textures/quartz_block.png:                  PNG image data, 16 x 16, 8-bit/color RGBA, non-interlaced
textures/quartz_chiseled.png:               PNG image data, 16 x 16, 8-bit/color RGBA, non-interlaced
textures/quartz_crystal_full.png:           PNG image data, 16 x 16, 8-bit/color RGBA, non-interlaced
textures/quartz_crystal_piece.png:          PNG image data, 16 x 16, 8-bit/color RGBA, non-interlaced
textures/quartz_ore.png:                    PNG image data, 16 x 16, 8-bit/color RGBA, non-interlaced
textures/quartz_pillar_side.png:            PNG image data, 16 x 16, 8-bit/color RGBA, non-interlaced
textures/quartz_pillar_side_horizontal.png: PNG image data, 16 x 16, 8-bit/color RGBA, non-interlaced
textures/quartz_pillar_top.png:             PNG image data, 16 x 16, 8-bit/color RGBA, non-interlaced
```

yep, looks all OK!
